### ISSUE TITLE
React Pod has been deprecated

### ISSUE THREAD

**Issue by emmby** [rebeccahughes/react-native-device-info#63]

**Comment by maciekiks** [comment]
workaround:
add to Podfile:
 pod 'React', :path => '../node_modules/react-native'

**Comment by koenpunt** [comment]
Is the Pod even necessary?

**Comment by jwrubel** [comment]
In addition to the pods @koenpunt mentions, pod yoga needs to be lower case:

`pod 'yoga', path: '../node_modules/react-native/ReactCommon/yoga'`

[This page](https://facebook.github.io/react-native/docs/integration-with-existing-apps.html) from React Native may help.

**Comment by Zephir77167** [comment]
Have had a lot of problems since adding this workaround in my podfile. Why exactly does this module has React as a cocoapod dependency?

**Comment by jwrubel** [comment]
React isn't a dependency of this pod, it's a dependency of React Native. I have found problems with a number of different libraries when I started using cocoapods. I saw on another issue list to add the following at  the end of your `Podfile`:

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == "React"
      target.remove_from_project
    end
  end
end
```
Once I added this post_install hook, everything got much more stable.

**Comment by machour** [comment]
@valerybugakov could you provide a PR for this?

**Comment by machour** [comment]
Fixed in 0.21.1

**Comment by wangwcq** [comment]
pod 'React', path: '../node_modules/react-native'
pod 'yoga', path: '../node_modules/react-native/ReactCommon/yoga'

For the pod 'yoga', use lower case package name or pod couldn't find podspec file.
